metrics/network/network-metrics-iperf3.sh

```diff
@@ -16,46 +16,41 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
-# Test inter-container network bandwidth and jitter using iperf3
+# This test will measure the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics will be got from the interconnection between a container
+# server and client using the iperf tool.
+# container-server <----> container-client
 
 set -e
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
-
-source "${SCRIPT_PATH}/lib/network-test-common.bash"
+source "${SCRIPT_PATH}/lib/network-common.bash"
+source "${SCRIPT_PATH}/../lib/common.bash"
 
 # Port number where the server will run
-port=5201:5201
+port="5201:5201"
 # Image name
-image=gabyct/network
+image="gabyct/network"
 # Measurement time (seconds)
-time=5
-# Name of the containers
-server_name="network-server"
-client_name="network-client"
-
-# Arguments to run the client
-if [ "$RUNTIME" == "runc" ];then
-	extra_args="-ti --privileged --rm"
-else
-	extra_args="-ti --rm"
-fi
-
-## Test name for reporting purposes
-test_name="network metrics iperf3"
-
-# This script will perform all the measurements using a local setup using iperf3
+transmit_time=5
+# Arguments to run the client/server
+client_extra_args="-ti --privileged --rm"
+server_extra_args="--privileged"
```

Comment:
I think I mentioned this before somewhere....
iirc, those `--priv` args are for non-CC runtimes only - but I suspect we are passing them in when running CC as well. Officially we do not support that arg (https://github.com/clearcontainers/runtime/blob/master/docs/limitations.md#docker-run---privileged) - and maybe we 'get away' with it right now as we might just ignore it.
I'd be happier if we didn't supply the arg to CC runtimes if possible - and also add a comment on why `--priv` is required above.